packages/google-maps/README.md

```diff
@@ -95,32 +92,39 @@ in `App_Resources/iOS/Info.plist`
 </plist>
 ```
 
+## Installation
+```cli
+npm install @nativescript/google-mapss
+```
+
 ## Usage
+To show the map view, add the `<MapView>` element to your HTML. Then to manage the mapping features, get the reference to the [GoogleMap](#googlemap-object) instance from the `ready` event data.
```

Comment:
@herefishyfish I think by mentioning `GoogleMap` here, we can leave its API reference where it is, to maintain the documentation structure of `Installation` -> `Usage` -> `API`. What do you think?

Reply:
Yeah I'd be happy with that if we can make it more pronounced, as it stands I can see users skipping over it and it's super important. Might also be better off after the flavours though as most people will likely skip to their flavour of choice.